### PR TITLE
LPS-173597 Cleanup in DLProcessorRegistryImpl

### PR DESCRIPTION
**Pull request by MrJohn1911**
Previous pull: https://github.com/liferay-core-infra/liferay-portal/pull/3234

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:dl-store

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/document-library/document-library-preview-audio/src/main/java/com/liferay/document/library/preview/audio/internal/AudioProcessorImpl.java [hunk 1]
```diff
@@ -42,14 +43,18 @@
...
 
 /**
  * @author Juan González
  * @author Sergio González
  * @author Mika Koivisto
  * @author Ivica Cardic
  */
+@Component(
+	property = "type=" + DLProcessorConstants.AUDIO_PROCESSOR,
+	service = {AudioProcessor.class, DLProcessor.class}
```
Comment (tinatian) [inline modules/apps/document-library/document-library-preview-audio/src/main/java/com/liferay/document/library/preview/audio/internal/AudioProcessorImpl.java:56]: @MrJohn1911 @rafaprax Why define two types of service here? I have the same question at https://github.com/liferay-core-infra/liferay-portal/pull/3234#issuecomment-1419649717

Comment (MrJohn1911) [reply modules/apps/document-library/document-library-preview-audio/src/main/java/com/liferay/document/library/preview/audio/internal/AudioProcessorImpl.java:56]: Hi @tinatian I already answered you [here](https://github.com/liferay-core-infra/liferay-portal/pull/3392#issuecomment-1422868834)

**Comment by tinatian** [comment]
@MrJohn1911 please try your best to remove the usages and make it only register as one type of service. 

Please resend.

Thanks.
Tina

**Pull request closed without merging**